src/renderers/webgl/WebGLProgram.js

```diff
@@ -209,6 +252,10 @@ THREE.WebGLProgram = ( function () {
 				parameters.alphaMap ? '#define USE_ALPHAMAP' : '',
 				parameters.vertexColors ? '#define USE_COLOR' : '',
 
+				parameters.energyConserving ? '#define ENERGY_PRESERVING_MONOCHROME' : '',
```

Comment:
You found the "bonus addition", if this is enabled, there is correct energyConserving behavior. :)